### ISSUE TITLE
Implement pipeline recommendations

### ISSUE THREAD

**Issue by tms-epcc** [lsst-uk/sky-estimation-WP3.7#58]
- Run the pipeline with required parameters
- ensure nothing breaks
- make sure resulting deep coadd looks ok
- this will iterate since almost certainly items will break.
- a few iterations will therefore be needed
- write report

**Comment by tms-epcc** [comment]
20/JUL/23
-  Aaron to get credentials for new system before starting

**Comment by tms-epcc** [comment]
09/AUG/23
- now have credentials
- able to run but crashing

**Comment by tms-epcc** [comment]
31/AUG/23
- Lee on vacation so no progress
- will review the documentation from the metrics workshop and if continues to crash, will contact Lee

**Comment by tms-epcc** [comment]
04/OCT/23
- in dialog with Lee 
-  slow going, takes 24 hours to run 
-  expect it will be Nov/Dec rather than Sep.

**Comment by tms-epcc** [comment]
18/JAN/24
- discussed at Princeton with DM team
- submitted recommendation and justification
- ran full pipeline using the recommendations.  Unfortunately, but not unsurprisingly, there are some items that appeared that need to be rectified.
- if unable to make recommendations part of default pipeline then the plan is to instead branch the pipeline.  The decision will depend on the timescales for when ComCam data starts arriving at then end of the year.  Want this to be part of the commissioning.  But decision will be made as late as possible since aim is to have a better product.  This is expected to also be DM strategy. 
- Aim is to have solution of some kind by early 25

**Comment by tms-epcc** [comment]
07/FEB/24
- Aaron looked at DM metrics and compared photometry metrics with default pipeline and the LSB recommended version
- so far preliminary results are promising and indicate no impact.
- Further metrics will be examined to further confirm this.
- If yes, will propose DM.

**Comment by tms-epcc** [comment]
14/MAR/24
- no progress, now in DM's control but they are busy with PSF subtraction
- Once that is sorted, expect will move onto sky subtraction

**Comment by tms-epcc** [comment]
11/APR/24
- discussion with DM team 
- investigating why patch boundaries are seen on stitching even with default pipeline
- Lee do not want to follow LSB team recommendation since it affects quality 
- Aaron has to investigate how to prevent artefacts appearing 
- Background match (the stretch goal)  is one further option so Aaron will investigate further.
- Aaron and Sugata will agree the way forward in the next few days
- This iterative interaction with DM is as expected.

**Comment by tms-epcc** [comment]
02/MAY/24
- submitted interesting plots re. patch boundaries but as yet no response

**Comment by tms-epcc** [comment]
23/MAY/24
- so this is being tracked in a separate ticket (#73) and the current ticket is closed
